source/OutfitterPanel.cpp

```diff
@@ -475,7 +475,8 @@ ShopPanel::BuyResult OutfitterPanel::CanBuy(bool onlyOwned) const
 		return errors[0];
 	else
 	{
-		string errorMessage = "There are several reasons why you cannot buy this outfit:\n";
+		string errorMessage = "There are several reasons why you cannot " +
+			string(onlyOwned ? "load" : "buy") + " this outfit:\n";
```

Comment:
```suggestion
			string(onlyOwned ? "install" : "buy") + " this outfit:\n";
```

To fit with the rest of this section. Also, `load` is ambiguous between installation, carriage as cargo, and loading from an external file.

Reply:
Oh I just used the same "load" phrase that was elsewhere. The outfitter is always confusing whether a button installs, moves, or buys, and from store, cargo or storage ...

Reply:
Huh...
I think where that snippet was used (OutfitterPanel.cpp, line 407), it was referring to cargo. Maybe something to change in a separate PR there.

Reply:
#10743 makes this OBE